### ISSUE TITLE
Header Images

### ISSUE THREAD

**Issue by josev814** [josev814/supercitymfg#5]
Will give us some photos to have cycle in addition to main building photo on home page
(Shawn explained this could be tricky as we had difficulty getting an image that worked well for the header that scaled well for mobile as well)

**Comment by josev814** [comment]
Waiting on images

**Comment by josev814** [comment]
I have this task on hold until we get images to upload for this.
